homeassistant/components/garmin_connect/__init__.py

```diff
@@ -21,7 +21,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORMS = ["sensor"]
-MIN_SCAN_INTERVAL = timedelta(minutes=10)
+MIN_SCAN_INTERVAL = timedelta(minutes=5)
```

Comment:
Why do we raise this?

Reply:
Because the new python library support this now, before this 10 minutes was minimum to prevent http toomanyrequests errors.

Reply:
I can remove it but have a new PR just for this is silly?

Reply:
Switched it back